packages/expo-video/ios/VideoPlayer.swift

```diff
@@ -160,18 +160,46 @@ internal final class VideoPlayer: SharedRef<AVPlayer>, Hashable, VideoPlayerObse
       try drm.type.assertIsSupported()
       contentKeyManager.addContentKeyRequest(videoSource: videoSource, asset: asset)
     }
-
+    // added player metadata, specifically for TV implementation
+    let metadata = Metadata(
+      title: videoSource.metadata?.title ?? "",
+      artist: videoSource.metadata?.artist ?? "",
+      subTitle: videoSource.metadata?.subTitle ?? ""
+    )
+    let metadataItems = createMetadataItems(for: metadata)
+    playerItem.externalMetadata = metadataItems
     playerItem.audioTimePitchAlgorithm = preservesPitch ? .spectral : .varispeed
     playerItem.preferredForwardBufferDuration = bufferOptions.preferredForwardBufferDuration
 
     // The current item has to be replaced from the main thread. When replacing from other queues
     // sometimes the KVOs will try to deliver updates after the item has been changed or player deallocated,
     // which causes crashes.
-    DispatchQueue.main.async { [weak self] in
-      self?.pointer.replaceCurrentItem(with: playerItem)
+    DispatchQueue.main.async { [pointer] in
+      pointer.replaceCurrentItem(with: nil)
     }
   }
 
+    func createMetadataItems(for metadata: Metadata) -> [AVMetadataItem] {
```

Comment:
```suggestion
  private func createMetadataItems(for metadata: Metadata) -> [AVMetadataItem] {
```
We can make it private